### ISSUE TITLE
hocr-clean

### ISSUE THREAD

**Issue by zuphilip** [tmbdev/hocr-tools#120]
Go through all ocr-elements and delete empty elements and possibly also elements with spaces only. Either do this recursive or start with the top elements and look at the `textContent`.

**Comment by amitdo** [comment]
This should be fixes in tesseract.

**Comment by stweil** [comment]
Yes, we think so, too. That would fix new hOCR files. Nevertheless hocr-clean would be useful for existing hOCR files.
